.travis/install-ivykis.sh

```diff
@@ -0,0 +1,17 @@
+#!/bin/bash -x -e
+
+OLDPWD=`pwd`
+
+cd $TRAVIS_BUILD_DIR
+curl -sL https://github.com/balabit/ivykis/archive/master.zip -o ivykis.zip
```

Comment:
For network operations, you could consider `travis_retry` as per https://blog.travis-ci.com/2013-05-20-network-timeouts-build-retries/

It should be exported in the main script, maybe you need to `export` it to work in subshells (I never tried).